tower: prepare to release 0.4.8 (#591)
tower-rs/tower: tower/CHANGELOG.md

# Unreleased

Nothing yet.

# 0.4.8 (May 28, 2021)

- **builder**: Add `ServiceBuilder::map_result` analogous to

>   `ServiceExt::map_result`.

Reviewer: nit: would be nice to reference a PR for this as well:
```suggestion
  `ServiceExt::map_result` ([#583]) 
```

> - **limit**: Add `GlobalConcurrencyLimitLayer` allowing to reuse concurrency
>   limit across multiple services ([#574])

Reviewer: grammar nit:
```suggestion
- **limit**: Add `GlobalConcurrencyLimitLayer` to allow reusing a concurrency
  limit across multiple services ([#574])
```

> [#574]: https://github.com/tower-rs/tower/pull/574

Reviewer: ```suggestion
[#574]: https://github.com/tower-rs/tower/pull/574
[#583]: https://github.com/tower-rs/tower/pull/583
```
